src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,418 @@
+module Algebra.Graph.Acyclic.AdjacencyMap (
+  -- * Data types and type aliases
+  PartialOrder,
+
+  -- * Data structure
+  AdjacencyMap,
+
+  -- * Internal Checks
+  consistent,
```

Comment:
Please be consistent with other modules, where this function comes last in the "Miscellaneous" section.